scripts/get-gee-build-environment.sh

```diff
@@ -32,14 +32,14 @@ MIN_GCC_VERSION="4.8"
 : ${PACKAGE_BUILD_DIR:=""}
 
 # Whether to delete the package build directory before exiting:
-: ${CLEAN_PACKAGE_BUILD_DIR:="yes"}
+: ${CLEAN_PACKAGE_BUILD_DIR:="no"}
```

Comment:
Is changing the default values necessary for building MrSID?

Reply:
When running the script on centos6, the build died before the MrSID script was even executing. It looked to be dying around the time it was getting to cleanup of the gtest rpm. It wasn't happening in any of the other platforms so I suggested turning that flag off and seeing if it had an effect.

I assume it must have had an effect. Were there any issues vis a vis centos6 initially? Rhel6?

I'm a bit confused as to why -c exists if it defaults to yes also...